grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -610,12 +643,16 @@ public GrpcService build() {
         if (USE_COROUTINE_CONTEXT_INTERCEPTOR) {
             final ServerInterceptor coroutineContextInterceptor =
                     new ArmeriaCoroutineContextInterceptor(useBlockingTaskExecutor);
+            interceptors.add(coroutineContextInterceptor);
```

Comment:
How about defining a private method like this to ensure `interceptors` is always non-null before adding something? This way, we can remove `if` you added in the constructor.

```java
private ImmutableList.Builder<...> interceptors() {
    if (interceptors != null) {
        return interceptors;
    } else {
        return interceptors = ImmutableList.builder();
    }
}
```